supplementary/alica_ros_turtlesim/alica/Expr/src/conditions/conditions.cpp

```diff
@@ -15,7 +15,7 @@ namespace alica
 bool conditionMove2Init748720375848597116(const Blackboard* input, const RunningPlan* rp, const IAlicaWorldModel* wm)
 {
     /*PROTECTED REGION ID(condition748720375848597116) ENABLED START*/
-    return turtlesim::ALICATurtleWorldModel::get()->getInit();
+    return true; // turtlesim::ALICATurtleWorldModel::get()->getInit(); // todo luca when ITransitionConditionCreator can be written dynamic
```

Comment:
so the conditions are always statically linked now (even if dynamic=true)? Are you planning to load them dynamically in this PR?